extras/cronwrapper

```diff
@@ -48,10 +48,10 @@ LOGFILE=$(mktemp /tmp/plexupdate.cron.XXXX)
 
 RET=0
 if $LOGGING; then
-	"${SCRIPT}" "${CONF}" 2>&1 | tee ${LOGFILE} | logger -t plexupdate -p daemon.info
+	"${SCRIPT}" --config "${CONF}" --notify-success 2>&1 | tee ${LOGFILE} | logger -t plexupdate -p daemon.info
```

Comment:
See line 44 above, this will result in `--config --config ${CONF}`.

Also, if notify-success is going to be an option, we shouldn't hard-code it into cronwrapper. We should rely on it being set in the config file. That way if someone doesn't want an email when an update is installed they just have to edit their config file, not cronwrapper itself.